webportal/src/app/cluster-view/hardware/hardware.component.js

```diff
@@ -17,14 +17,155 @@
 
 
 // module dependencies
+require('datatables.net/js/jquery.dataTables.js');
+require('datatables.net-bs/js/dataTables.bootstrap.js');
+require('datatables.net-bs/css/dataTables.bootstrap.css');
+require('datatables.net-plugins/sorting/natural.js');
+require('datatables.net-plugins/sorting/ip-address.js');
+require('datatables.net-plugins/sorting/title-numeric.js');
+require('jquery.ajax-cross-origin/js/jquery.ajax-cross-origin.min.js')
+const url = require('url');
 const hardwareComponent = require('./hardware.component.ejs');
+const breadcrumbComponent = require('../../job/breadcrumb/breadcrumb.component.ejs');
+const loading = require('../../job/loading/loading.component');
 const webportalConfig = require('../../config/webportal.config.json');
 
 const hardwareHtml = hardwareComponent({
+  breadcrumb: breadcrumbComponent,
   grafanaUri: webportalConfig.grafanaUri
 });
 
-$("#sidebar-menu--cluster-view").addClass("active");
-$("#sidebar-menu--cluster-view--hardware").addClass("active");
+/*
```

Comment:
Do we still need this if the corss origin header settled correctly?

Reply:
#Solved.